presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -198,6 +198,7 @@
     private int materializedViewMissingPartitionsThreshold = 100;
 
     private boolean verboseRuntimeStatsEnabled;
+    private boolean hudiEnabled = true;
```

Comment:
This property is enabled by default, so by default this would change the behavior rather than opt in ? Was that the intention ?

Reply:
Yes. We thought that keeping it enabled by default strikes a balance between seamless deployment and mitigating risk. If the reader path breaks, one can disable the config.

Reply:
If we don't have this by default, queries would return incorrect results by default. It'd be a large issues for many companies out there.